null_jobs_backend/settings.py

```diff
@@ -241,3 +242,53 @@
 
 # GOOGLE AUTH
 SITE_ID = 2
+
+LOG_DIR = os.path.join(BASE_DIR, "logs")
+
+if not os.path.exists(LOG_DIR):
+    os.makedirs(LOG_DIR)
+
+LOGGING = {
```

Comment:
Implement flag based file logging. 
introduce a variable, and based on its Boolean value i.e., true or false, if True, then record the logs into the file or else don't.